### DISCUSSION TITLE
Tweaking knob sensitivity via midi_value divisor

### DISCUSSION THREAD

**Discussion by buzkorian** [rsjaffe/MIDI2LR#566] [Q&A] [answered]
Sorry if this is poorly expressed, but I don't know all the right words.

I'm looking at the Behringer X-Touch Mini, which I understand has knobs which are not free-spinning but "click" for each fraction of a turn.  My understanding is that MIDI2LR assumes each click is intended to move the linked value (Temperature, Tint etc) by a set amount, being 1/127th of the range of the linked value.  [Another answer](https://github.com/rsjaffe/MIDI2LR/issues/224) says you can control the value of each click by limiting the range (minimum to maximum) of the linked value.  However, I understand this means that each click is still a set **proportion** of that range - that is, (maximum - minimum)/127.

If that's correct, would it also be possible to tweak the value of each click by using a different number in place of "127"?  A higher value for finer adjustments, or a lower value for coarser ones?

Is that number something users can set in the options?  Ideally you'd be able to adjust it per linked value like you can with min/max values.

**Comment by rsjaffe** [comment, marked as answer]
Regular controls only send 128 different values (0 through 127) so there's no way to have finer adjustments. It's a limitation of MIDI. There are two ways to get around it:
1. Make the control an NRPN control (some MIDI control surfaces allow that). Then you have 16,384 different values (0 through 16,383), so much finer control.
2. Make the control a differential control. Then MIDI2LR allows you to set the number of steps between min and max. This is because a differential control sends an up or down signal rather than sending an absolute value, so MIDI2LR can use an arbitrary scale.
